makefile: yosys 0.13 or newer required

oharboe opened the pull request:
use yosys that ships with OpenROAD if yosys is too old.

It is highly likely than e.g. Ubuntu has a too old Yosys installed, so this avoids a common pitfall & confusion when relying on ORFS to find the tools in the path, i.e. when not using ". env.sh"

For the normal use-case it is no longer necessary to run ". env.sh" with this change.

oharboe committed:
makefile: yosys 0.13 or newer required

use yosys that ships with OpenROAD if yosys is too old.

It is highly likely than e.g. Ubuntu has a too old Yosys
installed, so this avoids a common pitfall & confusion when relying
on ORFS to find the tools in the path, i.e. when not
using ". env.sh"

Signed-off-by: Øyvind Harboe <oyvind.harboe@zylin.com>

maliberty commented on flow/Makefile:
```diff
@@ -279,8 +279,15 @@ OPENROAD_NO_EXIT_CMD     = $(OPENROAD_EXE) $(OPENROAD_ARGS)
+ifeq ($(VERSION_GTEQ),0.13)
+  $(info Using Yosys $(YOSYS_VER) found in path, it is newer or equal to 0.13)
+else
   YOSYS_CMD              = $(abspath $(FLOW_HOME)/../tools/install/yosys/bin/yosys)
```
Why not just use the OR version unconditionally?

oharboe replied:
I was trying to retain the intent of the code any address only a specific concern.

same for openroad? ignore what is in the path and only if explicitly overriden use something else?

I like it. Simple. Less that can go wrong.

Will it work with the .deb installer?

oharboe committed:
makefile: do not use outdated yosys binaries

Signed-off-by: Øyvind Harboe <oyvind.harboe@zylin.com>

oharboe committed:
makefile: use yosys and openroad shipped with ORFS by default

YOSYS_CMD and/or OPENROAD_EXE must be explicitly specified to use
different yosys/openroad versions.

Signed-off-by: Øyvind Harboe <oyvind.harboe@zylin.com>
